### DISCUSSION TITLE
Sentry for self-hosted?

### DISCUSSION THREAD

**Discussion by Manouchehri** [langfuse/langfuse#1054] [Support] [answered]
For self-hosted Langfuse, is setting these undocumented Sentry env variables expected to work properly? =) 

```
NEXT_PUBLIC_SENTRY_DSN=
NEXT_SENTRY_ORG=
NEXT_SENTRY_PROJECT=
```

**Comment by Manouchehri** [comment, marked as answer]
Seems to work fine by just setting `NEXT_PUBLIC_SENTRY_DSN`. =)

**Reply by marcklingen** [reply]
This works as we use Sentry for Langfuse Cloud. However, this is not covered by semver and thus not included in the docs.

Use at your own risk, we might switch observability providers for this project down the road.
